### DISCUSSION TITLE
PCPI Z80 Support?

### DISCUSSION THREAD

**Discussion by kwadsten** [markadev/AppleII-VGA#41] [Q&A] [answered]
Hi, I built your card and it's working great.  I was wondering if it supports the PCPI/Z80/CPM firmware?  I assume not, but am not sure why it doesn't.

I tried it, but it didn't work.  Some of the other A2VGA cards seem to support this alternative firmware.   

https://github.com/V2RetroComputing/analog-firmware/releases/tag/20230406

**Comment by markadev** [comment]
Hey there, I'm glad it's working for you!  This card is just a VGA card so doesn't support the extra functions like Z80 emulation.  The V2-analog (and the modified version by Ralle Palaveev) is a fork of this project that added the extra functions and there are some hardware changes that make the Z80 emulation possible.

**Reply by kwadsten** [reply, marked as answer]
Thanks for the quick response. I also made a Ralle card a while back, and you are correct it does work with the Z80 code.

Hopefully you’ll consider updating your board to incorporate the necessary components. The simple design of your card and the nice VGA connector board make your version a slightly better design, in my opinion.
